helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -1725,6 +1718,19 @@ public HostStatus getHostStatus(final String host) {
         .build();
   }
 
+  @Override
+  public Map<String, String> getHostLabels(final String host) {
+    final ZooKeeperClient client = provider.get("getHostStatus");
+
+    if (!ZooKeeperRegistrarUtil.isHostRegistered(client, host)) {
+      log.warn("Host {} isn't registered in ZooKeeper.", host);
```

Comment:
would be helpful to have more context in log messages like this - like what method this is from, or what we were doing with this host in the first place, etc. In this case though is the log warning even useful?